src/Maker/MakeRegistrationForm.php

```diff
@@ -244,11 +304,53 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             $this->fileManager->dumpFile($classDetails->getPath(), $userManipulator->getSourceCode());
         }
 
+        if ($input->getArgument('will-verify-email')) {
+            $classDetails = new ClassDetails($userClass);
+            $userManipulator = new ClassSourceManipulator(
+                file_get_contents($classDetails->getPath())
+            );
+            $userManipulator->setIo($io);
+
+            $userManipulator->addProperty('verified', ['@ORM\Column(type="boolean")'], false);
+            $userManipulator->addAccessorMethod('verified', 'isVerified', 'bool', false);
+            $userManipulator->addSetter('verified', 'bool', false);
+
+            $this->fileManager->dumpFile($classDetails->getPath(), $userManipulator->getSourceCode());
+        }
+
         $generator->writeChanges();
 
         $this->writeSuccessMessage($io);
-        $io->text('Next: Go to /register to check out your new form!');
-        $io->text('Make any changes you need to the form, controller & template.');
+        $this->successMessage($io, $input->getArgument('will-verify-email'), $userClassNameDetails->getShortName());
+    }
+
+    private function successMessage(ConsoleStyle $io, bool $emailVerification, string $userClass): void
+    {
+        $closing[] = 'Next:';
+
+        if (!$emailVerification) {
+            $closing[] = 'Make any changes you need to the form, controller & template.';
+        } else {
+            $closing[] = '1) Set the redirect route on successful email verification in the <fg=yellow>RegistrationController</>';
+            $closing[] = '2) Display the success flash message in your redirect template, or remove the message all together.';
```

Comment:
```suggestion
            $closing[] = '2) A <fg=yellow>success</> flash message is added after verification. Be sure that you are rendering success flash messages or modify the flash code in <fg=yellow>RegistrationController:: verifyUserEmail ()</>';
```

This is WAY too long... but it *is* kinda what I want to say :p.

Reply:
done